06-Customizing AzTS for your org/Extending AzTS/Readme.md

```diff
@@ -2,19 +2,24 @@
 <br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
 
 # Extending AzTS
-Azure Tenant Security Solution (AzTS) provides capability to add new controls(for existing services supported by AzTS) to customize the AzTS for your organization as per your need.
-Currently you can extend AzTS controls set by either adding a new control based on custom Azure Policy or based on Microsoft Defender for Cloud assessment using Control Metadata Editor Tool. 
 
-## Enabling Control Medatadata Editor tool
-This feature is not enabled by default. To enable this feature for your AzTS setup, please follow steps mentioned [here](Prerequisites.md#prerequisite-azts-configurations-to-enable-control-medatadata-editor-toolcmet).
+## Customizing AzTS for your org
 
-## Add new control for existing services
+### Control Metadata Editor tool
+CMET (Control Metadata Editor Tool) is a tool that can be used by privileged users (mostly by admins maintaining AzTS setup for organization)to customize AzTS for organization. It provides capabilities to update control metadata (Tags, Display Name, control settings(if applicable)), enable or disable control, etc.
+This feature is disabled by default. To enable this feature for your AzTS setup, please follow steps mentioned [here](../Extending%20AzTS/Prerequisites.md#prerequisite-azts-configurations-to-enable-control-metadata-editor-toolcmet).
 
-   - [Add new control based on custom Azure policy](AddControlForPolicy.md)
-   - [Add new control based on MDC Assessment](AddControlForAssessment.md) 
+CMET (Control metadata editor tool) is only accessible to privileged users based on AzTS API configurations. Please follow the steps mentioned [here](../06-Customizing%20AzTS%20for%20your%20org/Extending%20AzTS/Prerequisites.md#access-to-cmet-control-metadata-editor-tool) to add yourself as privileged user.(This is only required once per user).
 
-## Update existing control metadata
-   - [Update existing control metadata](UpdateControlMetadata.md)
+Once CMET is enabled, you can use below features:
+
+   - [Add new control based on custom Azure policy](/06-Customizing%20AzTS%20for%20your%20org/Extending%20AzTS/AddControlForPolicy.md)
+   - [Add new control based on MDC Assessment](/06-Customizing%20AzTS%20for%20your%20org/Extending%20AzTS/AddControlForAssessment.md) 
+   - [Update control metadata for existing control](../06-Customizing%20AzTS%20for%20your%20org/Extending%20AzTS/UpdateControlMetadata.md)
+
+   If you are looking for updating multiple controls at once, please follow steps mentioned [here](../06-Customizing%20AzTS%20for%20your%20org/Extending%20AzTS/FeaturesInCMET.md#bulk-edit)
```

Comment:
If you need to update multiple controls at once, please follow the steps outlined here:

Reply:
Updated